src/comms/views.py

```diff
@@ -188,6 +195,58 @@ def news_list(request, tag=None):
     context = {
         'news_items': news_items,
         'tag': tag,
+        'all_tags': all_tags,
+    }
+
+    return render(request, template, context)
+
+
+def press_overview(request):
+    """
+    Gets NewsItem lists by tag or by the content type of the
+    object (e.g. press, journal), in the order they are passed in the URL.
+    Expects GET request query parameters such as in this example:
+    /news/press_overview/?tags__text=Announcement&tags__text=Blog&content_type__model=journal
+    :param request: HttpRequest object
+    :return: HttpResponse object
+    """
+
+    all_tags = models.Tag.objects.all().annotate(
+        Count('tags')
+    ).order_by('-tags__count', 'text')
+
+    tag_texts = [tag.text for tag in all_tags]
+    tag_sets = {}
+    object_sets = {}
+    for key in request.GET:
+        if key == 'tags__text':
+            for value in request.GET.getlist(key):
+                if value in tag_texts:
+                    tag_sets[value] = models.NewsItem.active_objects.filter(
+                        content_type=request.model_content_type,
+                        object_id=request.site_type.id,
+                        tags__text=value,
+                    )
+        elif key == 'content_type__model':
+            for value in request.GET.getlist(key):
+                if value in ['press', 'journal']:
+                    object_sets[value] = models.NewsItem.active_objects.filter(
+                        content_type__model=value,
+                    )
+
+    pinned_items = models.NewsItem.active_objects.filter(
+        content_type=request.model_content_type,
+        object_id=request.site_type.id,
+        pinned=True,
+    )
+
+    template = 'press/news/overview.html'
```

Comment:
I can't find this template neither in the codebase nor this PR, is it coming in another PR?

Reply:
It's intended to pick this up: https://github.com/BirkbeckCTP/hourglass/blob/main/templates/press/news/overview.html

Should I add a comment that it is only supported on the hourglass theme?

Reply:
It is worth adding a very basic template for OLH theme to avoid confusion

Reply:
Either solution is better than having a view request a template that doesn't exist. If we don't see use for this view outside of hourglass, then it is fine to have it there.

Thoughts @ajrbyers

Reply:
EIther works for me. In the short term a component is fine but we should consider extending some of the Hourglass features onto presses in the future.

Reply:
Great. Let's move it to a component for now. 

I will extricate it from this PR and then re-request review so we can keep the other improvements here.